lib/runner.js

```diff
@@ -65,6 +65,10 @@ optionChain(chainableMethods, function (opts, title, fn) {
 		throw new TypeError('Expected a function. Use `test.todo()` for tests without a function.');
```

Comment:
Instead of adding a new error message, why don't we just remove `opts.skipped && ` from here? Unless I'm missing something? // @novemberborn